input/pagecontent/terminology.md

```diff
@@ -1,35 +1,154 @@
+- [Medication request intent](https://hl7.org/fhir/R4/valueset-medicationrequest-intent.html)
+- [Medication status codes](https://hl7.org/fhir/R4/valueset-medication-statement-status.html)
+- [Medication usage category codes](https://hl7.org/fhir/R4/valueset-medication-statement-category.html)
+- [Medicationrequest status](https://hl7.org/fhir/R4/valueset-medicationrequest-status.html)
+- [NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)
+- [Observation Category Codes](https://hl7.org/fhir/R4/valueset-observation-category.html)
+- [Observation Interpretation Codes](https://hl7.org/fhir/R4/valueset-observation-interpretation.html)
+- [Observation Reference Range Meaning Codes](https://hl7.org/fhir/R4/valueset-referencerange-meaning.html)
+- [ObservationStatus](http://hl7.org/fhir/R4/valueset-observation-status.html)
+- [Participant type](https://hl7.org/fhir/R4/valueset-encounter-participant-type.html)
+- [Provenance activity type](https://hl7.org/fhir/R4/valueset-provenance-activity-type.html)
+- [Vital Signs Units](https://hl7.org/fhir/R4/valueset-ucum-vitals-common.html)
+- [VitalSigns](https://hl7.org/fhir/R4/valueset-observation-vitalsignresult.html)
+
 
 
 ### Code systems
 
-See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-systems.html) for a complete discussion on code systems and a list of code system names used in FHIR.  All code systems used in this guide are defined in the base FHIR specification or [AU Base](https://build.fhir.org/ig/hl7au/au-fhir-base/terminology.html). 
+See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-systems.html) for a complete discussion on code systems and a list of code system names used in FHIR. All code systems used in this guide are defined in the base FHIR specification or [AU Base](https://build.fhir.org/ig/hl7au/au-fhir-base/terminology.html). 
 
 No code systems have been defined for this implementation guide.
 
+
+**Code systems published in AU Base**
+- [ActCodeAU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-v3-ActCode.html)
+- [DegreeLicenseCertificate AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-v2-0360.html)
+- [IdentifierType AU](https://build.fhir.org/ig/hl7au/au-fhir-base//CodeSystem-au-v2-0203.html)
+- [Individual Healthcare Identifier Record Status](https://healthterminologies.gov.au/fhir/CodeSystem/ihi-record-status-1)
+- [Individual Healthcare Identifier Status](https://healthterminologies.gov.au/fhir/CodeSystem/ihi-status-1)
+- [Location Type (Physical) AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-location-physical-type.html)
+- [Location Type AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-location-type.html)
+- [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-pbs-item-external.html)
+- [providerRole AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-v2-0443.html)
+
+**Code systems published in the NCTS**
```

Comment:
Added

Reply:
I think LOINC is published in NCTS.

Reply:
For SNOMED. Maybe https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=snomed is a better link? But I'm not sure.

Reply:
LOINC 2.77 is out now...did we want to tie us to a version like that?

Reply:
Re: loinc. No we don't. It would be great if we could have a url that resolves to the latest. I'm probably less concerned for this March snapshot. But we definitely need a non-version specific link for the August ballot.

Reply:
> @mjosborne1 Re: loinc.... But we definitely need a non-version specific link for the August ballot.
@dt-r I'll talk to some of the ncts tooling perole about that.

Reply:
Actually, this already works
https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=loinc

Reply:
It filters to the most recent version of loinc

Reply:
1. SNOMED link changed to  https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=snomed 
2. LOINC moved to Code systems published in NCTS, and hyperlink changed to https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=loinc